PocketKit/Sources/PocketKit/Article/Highlights/HighlightsView.swift

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Localization
+import SwiftUI
+import Textile
+
+class HighlightsViewController: UIHostingController<HighlightsView> {
+    convenience init(highlights: [HighlightedQuote], viewModel: SavedItemViewModel) {
+        self.init(rootView: HighlightsView(highlights: highlights, viewModel: viewModel))
+    }
+
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        guard traitCollection.userInterfaceIdiom == .phone else { return .all }
```

Comment:
Why are we only allowing portrait on phone?

Reply:
It's a copy/paste leftover, and yes we should also allow landscape since we do on reader anyway, thanks for spotting it!